src/pane.coffee

```diff
@@ -514,7 +514,8 @@ class Pane extends Model
 
     saveOptions = item.getSaveDialogOptions?() ? {}
     saveOptions.defaultPath ?= item.getPath()
-    newItemPath = atom.showSaveDialogSync(saveOptions)
+    showSaveDialogSync = if item.showSaveDialogSync else atom.showSaveDialogSync
```

Comment:
Yeah, this line of code doesn't look legit to me. It would have to use the `?` operator as you said. Also it would need to be bound to the item...

```coffee
showSaveDialogSync = item.showSaveDialogSync?.bind(item) ? atom.showSaveDialogSync.bind(atom)
```

Also, we'd need tests for this. I'd be open to adding a point of flexibility here but we need tests.